CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## Version 1.23.0
+* Release date: October 14, 2020
+* Release status: General Availability
+* Added deployments of Azure SQL DB and VM
+* Added PowerShell kernel results streaming support
+* Added improvements to SQL Database Projects extension
```

Comment:
BDC HDFS support for non-admin SQL logins

Reply:
These are the notes Alan used, so I'm gonna let them be to stay consistent